src/EditorFeatures/Core/GoToDefinition/DefinitionItemService.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Editor.FindUsages;
+using Microsoft.CodeAnalysis.Editor.Shared.Utilities;
+using Microsoft.CodeAnalysis.FindUsages;
+using Microsoft.CodeAnalysis.Host;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Shared.Extensions;
+
+namespace Microsoft.CodeAnalysis.Editor.GoToDefinition
+{
+    internal abstract class AbstractDefinitionItemService : ForegroundThreadAffinitizedObject, IDefinitionItemService, ILanguageService
+    {
+        public async Task<(string, ImmutableArray<DefinitionItem>)> GetDefinitionsAsync(Document document, int position, bool thirdPartyNavigationAllowed, CancellationToken cancellationToken)
```

Comment:
It's the name of the symbol as it should be presented if we have to use the FAR window--wasn't sure the best way to get it given this interface needs to not mention Symbols.